code/sarix_model/utils.py

```diff
@@ -0,0 +1,131 @@
+# This is a copy of ../gbq/utils.py
+# with updated model_name choices
+# In a future refactor, we should consolidate
+
+import argparse
+import importlib
+from pathlib import Path
+from types import SimpleNamespace
+
+import datetime
+
+def parse_args():
+    '''
+    Parse arguments to the sarix_model.py script
+    
+    Returns
+    -------
+    Two configuration objects collecting settings for the model and the run:
+    - `model_config` contains settings for the model
+    - `run_config` contains the following properties:
+        - `ref_date`: the reference date for the forecast
+        - `output_root`: `pathlib.Path` object with the root directory for
+            saving model outputs
+        - `max_horizon`: integer, maximum forecast horizon relative to the
+            last observed data
+        - `q_levels`: list of floats with quantile levels for predictions
+        - `q_labels`: list of strings with names for the quantile levels
+    '''
+    parser = _make_parser()
+    args = parser.parse_args()
+    
+    ref_date = _validate_ref_date(args.ref_date)
+    model_name = args.model_name
+    
+    model_config = importlib.import_module(f'configs.{model_name}').config
```

Comment:
If the args provide everything you need to know to construct the model_config, would recommend avoiding a dynamic import because those don't surface issues in the underlying module until runtime (i.e., static type checkers and other helpful IDE tools don't work with dynamic imports).

For example, you could have a helper function that returns the correct model_config (which I suspect will also make writing tests easier, if that's your plan)